src/pages/Tokens/index.tsx

```diff
@@ -56,12 +61,18 @@ const TokenItem = ({ token, isLast }: { token: State.UDT; isLast?: boolean }) =>
           <img src={token.iconFile ? token.iconFile : SUDTTokenIcon} alt="token icon" />
           <div>
             <TokensItemNamePanel>
-              {token.fullName ? (
-                <Link to={`/sudt/${token.typeHash}`}>{`${name} (${symbol})`}</Link>
+              {name ? (
+                <Link to={`/sudt/${token.typeHash}`}>
+                  {symbol}
+                  <span className={styles.name}>{name || defaultName}</span>
```

Comment:
`name` will not be null at this branch

Reply:
Fixed by [ff2a79e](https://github.com/nervosnetwork/ckb-explorer-frontend/pull/1033/commits/ff2a79e948569ee5ee2f2076e573d1e7ad40c355)